frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card-item.tsx

```diff
@@ -7,10 +7,13 @@ import {
   LoadingInline,
 } from '@console/internal/components/utils';
 
-const ItemBody: React.FC<ItemBodyProps> = React.memo(({ title, children }) => (
+const ItemBody: React.FC<ItemBodyProps> = React.memo(({ title, text, infoText }) => (
   <div className="co-inventory-card__item">
     <div className="nb-object-data-reduction-card__row-title">{title}</div>
-    {children}
+    <div className="nb-object-data-reduction-card__row-status-item">
+      {text}
```

Comment:
```
<span className="nb-object-data-reduction-card__row-status-item-text">
          {text}
</span>
```

Reply:
`text` is `Unavailable` and  `<LoadingInline/>`  and they have custom css as well
I would rather generalize them all up!
Thanx!